examples/json-wrapper-ext/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+// We use `JsonObject` in our UDL. It moves to and from Uniffi bindings via a string.
+pub type JsonObject = serde_json::Value;
```

Comment:
I think the type alias is more just to have a nice name for the UDL file.  The `FFIConverter` code is the key to avoiding the orphan rule.

Reply:
IIUC, `FfiConverter` is only part of the story - I believe that alone would have allowed us to create the `Handle` type, but would not quite have got `JsonObject` over the line - that's why the [UniffiCustomTypeWrapper trait](https://github.com/mozilla/uniffi-rs/blob/main/uniffi_bindgen/src/scaffolding/templates/ExternalTypesTemplate.rs#L14) is a private trait in the generated Rust scaffolding rather than being in uniffi itself (which is where `FfiConverter` is defined as a `pub trait`)

Reply:
(huh - so that's just a kinda "trait trampoline" to sidestep that rule? 🤔 )